docs/sources/enterprise/access-control/fine-grained-access-control-references.md

```diff
@@ -35,16 +35,16 @@ The reference information that follows complements conceptual information about
-| `fixed:current.org:reader`             | `orgs:read`<br>`orgs.quotas:read`                                                                                                                                                                                                                                        | Read the current organization, such as its ID, name, address, or quotas.                                                                                                                                                                                                              |
-| `fixed:current.org:writer`             | All permissions from `fixed:current.orgs:reader` and <br> `orgs:write`<br>`orgs.quotas:write`<br>`orgs.preferences:read`<br>`orgs.preferences:write`                                                                                                                     | Read the current organization, its quotas, or its preferences. Update the current organization properties, or its preferences.                                                                                                                                                        |
+| `fixed:organization:reader`            | `orgs:read`<br>`orgs.quotas:read`                                                                                                                                                                                                                                        | Read an organization and its quotas.                                                                                                                                                                                                                                                  |
+| `fixed:organization:writer`            | All permissions from `fixed:organization:reader` and <br> `orgs:write`<br>`orgs.preferences:read`<br>`orgs.preferences:write`                                                                                                                                            | Read an organization, its quotas, or its preferences. Update organization properties, or its preferences.                                                                                                                                                                             |
+| `fixed:organization:manager`           | All permissions from `fixed:organization:reader` and <br> `orgs:write`<br>`orgs:create`<br>`orgs:delete`<br>`orgs.quotas:write`                                                                                                                                          | Create, read, write, or delete an organization. Read or write its quotas. This role needs to be assigned globally.                                                                                                                                                                    |
```

Comment:
We added `manager`? :)

Reply:
Not acted yet, I'm getting ahead: https://github.com/grafana/grafana/pull/42049

Reply:
Got it, thank you.